nomenclature/core.py

```diff
@@ -28,20 +28,20 @@ def process(
 
     Parameters
     ----------
-    df : pyam.IamDataFrame
-        Input data to be validated and aggregated.
-    dsd : DataStructureDefinition
+    df : :class:`pyam.IamDataFrame`
+        Scenario data to be validated and aggregated.
+    dsd : :class:`DataStructureDefinition`
         Codelists that are used for validation.
     dimensions : list, optional
         Dimensions to be used in the validation, defaults to all dimensions defined in
         *dsd*
-    processor : RegionProcessor, optional
-        Region processor that will perform region renaming and aggregation if provided
+    processor : :class:`RegionProcessor`, optional
+        Region processor to perform region renaming and aggregation (if given)
```

Comment:
"(if given)" possible redundant since `processor` is "optional"

Reply:
Yes, redundant, but on purpose, because users may overlook the "optional" tag